composer.json

```diff
@@ -41,8 +41,8 @@
     "require-dev": {
         "doctrine/coding-standard": "9.0.0",
         "jetbrains/phpstorm-stubs": "2021.1",
-        "phpstan/phpstan": "0.12.96",
-        "phpunit/phpunit": "^7.5.20|^8.5|9.5.5",
+        "phpstan/phpstan": "0.12.99",
+        "phpunit/phpunit": "^7.5.20|^8.5|^9.5.5",
```

Comment:
I think we really mean to pin tools to a specific version, IMO you should drop that caret.

Reply:
Oh okay. I did expect pinning to be necessary for PHPUnit, but why not. Pinned again.